pkgs/applications/misc/yubioath-flutter/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, flutter
+, python3
+, fetchFromGitHub
+, stdenv
+, pcre2
+}:
+
+let
+  vendorHashes = {
+    x86_64-linux = "sha256:0bwrybwaq9q16dwr56590k8j66qfckrjihsqj9brwjvlqc1mc207";
+  };
+
+  version = "6.0.2";
+  src = fetchFromGitHub {
+    owner = "Yubico";
+    repo = "yubioath-flutter";
+    rev = "${version}";
```

Comment:
```suggestion
    rev = version;
```

Reply:
Good catch - I thought this was tagged as v${version} originally and forgot to unquote it once I noticed it wasn't :)